raft/src/test/java/org/apache/kafka/raft/RaftEventSimulationTest.java

```diff
@@ -502,7 +502,7 @@ int randomNodeId() {
         }
 
         Set<Integer> voters() {
-            return nodes.keySet();
+            return new HashSet<>(voters);
```

Comment:
nit: is the copy needed? I wouldn't typically care in a test case, but these ones are pretty cpu-expensive already.

Reply:
Yeah I agree